Gemfile

```diff
@@ -0,0 +1,11 @@
+source 'https://rubygems.org'
+
+# Specify your gem's dependencies in better_env.gemspec
+gemspec
+
+group :development, :test do
+  gem 'pry-byebug'
+  gem 'rubocop'
+  gem 'fasterer'
```

Comment:
What is this one for?

Reply:
Just a small tool for developers: https://github.com/DamirSvrtan/fasterer